cloud/kubernetes/helm/yugabyte/values.yaml

```diff
@@ -4,13 +4,13 @@
 Component: "yugabytedb"
 Image:
   repository: "yugabytedb/yugabyte"
-  tag: latest
```

Comment:
Can this be overridden by the helm chart?

Reply:
yes